gtfs/feed_sources/Berlin.py

```diff
@@ -0,0 +1,9 @@
+"""Fetch Berlin feed."""
+from utils.FeedSource import FeedSource
+
+class Berlin(FeedSource):
```

Comment:
We can make `FeedSource` a abstract base class, which mostly defines some needed attributes for all its derivatives, e.g.

```
from abc import ABC, abstractmethod

class FeedSource(ABC):
    
     @property
     @abstractmethod
     def url(self):
          pass
    
     @abstractmethod
     def fetch(self):
           pass
```

Then once you subclass `FeedSource` you can put `url` as a class attribute:

```
class Berlin(FeedSource):
    url = 'https..'
    
    def fetch(self):
        ...
```

The advantage is that Python won't allow subclasses which don't define the things marked as `abstractmethod`.